### ISSUE TITLE
Can only fetch first run id from cache using `SQLiteMetadataStore` object

### ISSUE THREAD

**Issue by elutins** [DAGWorks-Inc/hamilton#1204]
Cache only returns the first `run_id` that populated the cache when calling `get_run_ids()` on a `hamilton.caching.stores.sqlite.SQLiteMetadataStore` object

# Current behavior

- When calling `get_run_ids()` on a `hamilton.caching.stores.sqlite.SQLiteMetadataStore` class object, only the first `run_id` that populated the cache is returned rather than all `run_ids` of the cache. 

## Stack Traces

Believe this is caused by [this line here](https://github.com/DAGWorks-Inc/hamilton/blob/09f07129f2f7e6c06ec61914ca97e8df3dd08ed5/hamilton/caching/stores/sqlite.py#L180) in `hamilton/caching/stores/sqlite.py` in which the return `result` object is being indexed to the first item only

## Steps to replicate behavior
1. instantiate a driver object using `with_cache()` 
```
driver = (
    hamilton.driver.Builder()
    .with_config()
    .with_modules({some_module})
    .with_cache()
    .build()
)
```
2. execute the driver `driver.execute(final_vars=[some_var], inputs={some_input})`
3. execute the driver again grabbing a different `final_var`:  `driver.execute(final_vars=[some_other_var], inputs={some_input})`
4. run ` hamilton.caching.stores.sqlite.SQLiteMetadataStore(".hamilton_cache").get_run_ids()` to get the run_ids that populated cache
    - the expected/wanted behavior is that this command would return _all_ run_ids - not just the first 

## Library & System Information
- Using hamilton version `1.80.0` & python `3.12.7`

# Expected behavior

- When calling `get_run_ids()` on a `hamilton.caching.stores.sqlite.SQLiteMetadataStore` class object, I'd expect a full list of run ids that have populated the cache to be returned rather than just the first run.

# Additional context

**Comment by skrawcz** [comment]
@elutins thanks for flagging. We'll get a fix out today.

**Comment by skrawcz** [comment]
@elutins this has been fixed in `sf-hamilton==1.81.2`.  Thanks for raising!
